cmd/descheduler/app/options/options.go

```diff
@@ -58,7 +60,17 @@ func NewDeschedulerServer() (*DeschedulerServer, error) {
 }
 
 func newDefaultComponentConfig() (*componentconfig.DeschedulerConfiguration, error) {
-	versionedCfg := v1alpha1.DeschedulerConfiguration{}
+	versionedCfg := v1alpha1.DeschedulerConfiguration{
+		LeaderElection: componentbaseconfig.LeaderElectionConfiguration{
+			LeaderElect:       false,
+			LeaseDuration:     metav1.Duration{Duration: 15 * time.Second},
+			RenewDeadline:     metav1.Duration{Duration: 10 * time.Second},
+			RetryPeriod:       metav1.Duration{Duration: 2 * time.Second},
```

Comment:
This will generate a lot of requests to the apiserver. What about to go with LeaseDuration=137s, RenewDealine=107s, RetryPeriod=26s by default? Those are the defaults we use in OpenShift by experience.

Reply:
Thanks, updated. We forgot to recheck these values whether these appropriate or not. 

Did you use any calculation formula to get these values? For example, why `137` instead of `120` (which is 2m)?

Reply:
https://github.com/openshift/enhancements/blob/master/CONVENTIONS.md#handling-kube-apiserver-disruption will help with the context though it is not a must. We can go lower though I would stick with at least 60s for the renew deadline and lease duration.